docs/first_objective.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>Not sure if the values are 0 or too small for a few species. If they are 0, you don't need to plot them </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>They're not zero, but very small (as you can see from the newly printed dataframes. Maybe I should add some threshold, below which the ion wouldn't be plotted?</p>

Reply:
<p>Done, changed the y-scale to log</p>

Reply:
done